Userland/Libraries/LibGUI/AbstractView.cpp

```diff
@@ -663,12 +679,7 @@ void AbstractView::set_searchable(bool searchable)
         return;
     m_searchable = searchable;
     if (!m_searchable)
-        cancel_searching();
-}
-
-bool AbstractView::is_highlighting_searching(const ModelIndex& index) const
```

Comment:
More east const. If you're working on this file, you might as well improve all of them.